templates/SilverStripe/CMS/Search/Includes/SearchForm.ss

```diff
@@ -9,7 +9,7 @@
             placeholder="<%t CWP_Search.Placeholder "Search..." %>"/>
         <label class="sr-only" for="SearchForm_SearchForm_Search"><%t CWP_Search.Button "Search" %></label>
         <div class="input-group-append">
-            <button type="submit" class="btn btn-secondary btn-lg"><%t CWP_Search.Button "Search" %></button>
+            <button type="submit" class="btn btn-secondary btn-lg search__btn"><%t CWP_Search.Button "Search" %></button>
```

Comment:
Interestingly, if you remove the search__btn class and replace btn-secondary with btn-primary you get the styling you were after. I wonder if this should be changed to a primary button.